scripts/globals/effects/visitant.lua

```diff
@@ -6,7 +6,7 @@ require("scripts/globals/zone")
 -----------------------------------
 local effectObject = {}
 
-local remainingTimeLimits = { 300, 120, 60, 30, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 }
+local remainingTimeLimits = { 300, 240, 180, 120, 60, 30, 10, 5, 4, 3, 2, 1 }
```

Comment:
Can you confirm that this is the behavior on retail for updates?  In original testing, the previous version was what I observed.

Reply:
![image](https://user-images.githubusercontent.com/1618002/232209647-68d48de2-9a1a-48e9-aacc-0dd120206110.png)
This is a capture i made on retail 10 minutes ago